README.md

```diff
@@ -48,8 +48,14 @@ the path to get to `completed` is longer than the `submitted` one.
 The endpoint `/healthcheck` checks the connection to the database and returns
 one of:
 
-- success (200, JSON "status": "OK")
-- failure (500, JSON "status": "error").
+* success (200, JSON "status": "OK")
+* failure (500, JSON "status": "error").
+
+### Kafka scripts
+
+Contains tools to update and debug Kafka
```

Comment:
I was envisaging this more as a sentence - where the link is part of the sentence describing it. It's fine to keep this documentation quite conversational. Like these examples:
https://github.com/ONSdigital/dp-ci#adding-new-repositories-to-ci
https://github.com/ONSdigital/dp-ci#jenkins-orand-concourse-isnt-working